api/endpoint-tests/activities/approaches/put.js

```diff
@@ -56,11 +56,22 @@ tap.test(
         );
 
         authenticated.test(
-          'with approaches that are not an array',
+          'with approaches that are invalid',
           async invalidTest => {
             const { response, body } = await request.put(url(4100), {
               jar: cookies,
-              json: { hello: 'world' }
```

Comment:
Because of the way the `synchronize` method works, a non-array will actually work - `synchronize` will wrap it in an array if necessary.